open-machine-learning-jupyter-book/assignments/ml-advanced/clustering/introduction-to-clustering.md

```diff
@@ -0,0 +1,11 @@
+# Research other visualizations for clustering
+
+## Instructions
+
+In this lesson, you have worked with some visualization techniques to get a grasp on plotting your data in preparation for clustering it. Scatterplots, in particular are useful for finding groups of objects. Research different ways and different libraries to create scatterplots and document your work in a notebook. You can use the data from this lesson, other lessons, or data you source yourself (please credit its source, however, in your notebook). Plot some data using scatterplots and explain what you discover.
+
+## Rubric
```

Comment:
You need to add the acknowledge section. E.g. https://open-academy.github.io/machine-learning/assignments/data-science/small-diabetes-study.html#acknowledgments